twisted/twisted
# [9445] Re-enable macOS on circleCI.

Scope
======

See https://twistedmatrix.com/trac/ticket/9445

This re-enabled macOS builds on circleCI after circelCI donated more build times for Twisted.

With macOS on CircleCI we can run macOS tests from branches which don't belong to Twisted repo and which are not run on Buildbot.


Changes
=======

I have now enabled the macOS builds only after static checkers and documentation is green.
I hope that this can reduce the time we use macOS on circleCI 

How to test
=========

Check that this branch has macOS build https://circleci.com/gh/twisted/workflows/twisted

One build for this PR https://circleci.com/workflow-run/385ec102-6182-407f-9daa-ba3b4406377c
